### ISSUE TITLE
A11y: Focus design of the “Close menu” button in the mobile view not sufficient

### ISSUE THREAD

**Issue by blacksheep--** [shopware/platform#7092]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

The changed background color of the focus and hover highlighting of the “Close menu” button is too faint compared to its regular state. 
This makes it difficult to use with the keyboard, which is particularly challenging for people with a visual impairment.

*_Expected behaviour:_*

Make sure that the focus and hover design is not only based on color, but also includes another graphic change. This can be, for example, a dark background and the inversion of the font color (white font, dark background). A pure color change is only permitted as an exception if the contrast ratio of the two states (normal <> focus) is at least 3:1.

In addition, the contrast ratio to the surroundings must be sufficient even when the focus is activated. Normal text 4.5:1

*_How to reproduce:_*

Enable `ACCESSIBILITY_TWEAKS` Feature-Flag

1. open the menu in mobile view

![Image](https://github.com/user-attachments/assets/ab0847d5-582f-4c10-85d1-e79998f655c3)

![Image](https://github.com/user-attachments/assets/222c0d5b-701e-4614-856d-56af23dc6c4e)

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40846**



[created from NEXT-40846, comment 541825]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40846, comment 541999]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.3**

 This issue will be / was solved in Shopware version ***6.6.10.3***.



[created from NEXT-40846, comment 544068]
